atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/metrics/SweepMetricsAssert.java

```diff
@@ -115,15 +141,15 @@ public void hasNotRegisteredTargetedOutcome(SweepStrategy strategy, SweepOutcome
         objects.assertNull(info, getGaugeForTargetedOutcome(strategy, outcome));
     }
 
-    private Gauge<Long> getGaugeConservative(String name) {
+    private <N> Gauge<N> getGaugeConservative(String name) {
```

Comment:
I think this should be `private Gauge<?> getGaugeConservative(String name)`, it doesn't look like the type param is particularly helpful here

Reply:
👍